components/list/list-item-mixin.js

```diff
@@ -10,10 +11,23 @@ import { ifDefined } from 'lit-html/directives/if-defined.js';
 import { ListItemCheckboxMixin } from './list-item-checkbox-mixin.js';
 import { ListItemDragDropMixin } from './list-item-drag-drop-mixin.js';
 import { ListItemRoleMixin } from './list-item-role-mixin.js';
+import { LocalizeCoreElement } from '../../lang/localize-core-element.js';
 import { nothing } from 'lit-html';
 import ResizeObserver from 'resize-observer-polyfill';
 import { RtlMixin } from '../../mixins/rtl-mixin.js';
 
+let tabPressed = false;
+document.addEventListener('keydown', e => {
+	if (e.keyCode !== 9) return;
+	tabPressed = true;
+});
+document.addEventListener('keyup', e => {
+	if (e.keyCode !== 9) return;
+	tabPressed = false;
+});
```

Comment:
I don't think it's a big deal, but these event listeners will get added as soon as this file is included in the page, regardless of whether the page actually has a list or list in grid mode.

The only optimization I can think would be to run this code from the `connectedCallback` and gate it so that it only runs once -- it'd still run for non-grid mode but it's a slight improvement. Something like:

```javascript
let tabPressed = false;
let tabListenerAdded = false;
function addTabListener() {
	if (tabListenerAdded) return;
	tabListenerAdded = true;
	document.addEventListener('keydown', e => {
		if (e.keyCode !== 9) return;
		tabPressed = true;
	});
	document.addEventListener('keyup', e => {
		if (e.keyCode !== 9) return;
		tabPressed = false;
	});
}
```

Reply:
That's a good point.  I will make that improvement.